### PR TITLE
Add Dockerfile& instrcutions& modify CST pipelines related to pyre

### PR DESCRIPTION
**Pull request by LangFeng0912**
I added the Dockerfile, and the instructions for that;
moreover, the new version of pyre needs to init and start manually, and therefore the CST_pipeline of libsa4py needs a bit change.

### CODE REVIEW

File: libsa4py/run_process.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import subprocess
+from typing import Dict, List, Mapping, Optional, Sequence, Tuple
+
+
+def run(
```
Comment (mir-am) [inline libsa4py/run_process.py]: Please move this `run()` to `utils.py`.

File: libsa4py/cst_pipeline.py [hunk 2]
```diff
@@ -160,6 +160,7 @@ def process_project(self, i, project):
             if len(project_files) != 0:
                 if self.use_pyre:
                     print(f"Running pyre for {project_id}")
+                    # therefore we need to start pyre manually
```
Comment (mir-am) [inline libsa4py/cst_pipeline.py]: Please remove this comment.

File: README.md [hunk 3]
```diff
@@ -65,4 +65,7 @@ Description:
 - `--o $OUTPUT_PATH`: Path to the processed projects, used in the previous processing step.
 
 # JSON Output
-After processing each project, a JSON-formatted file is produced, which is described [here](https://github.com/saltudelft/light-sa-type-inf/blob/master/JSONOutput.md).
\ No newline at end of file
+After processing each project, a JSON-formatted file is produced, which is described [here](https://github.com/saltudelft/light-sa-type-inf/blob/master/JSONOutput.md).
+
+# Docker options
```
Comment (mir-am) [inline README.md]: `Docker options` -> `Docker image`

Comment (LangFeng0912) [reply README.md]: renamed

**Pull request merged**